contributors/devel/help-wanted.md

```diff
@@ -1,3 +1,3 @@
 This document has moved [here](https://git.k8s.io/community/tree/master/contributors/guide/help-wanted.md).
 
-*This file is a redirect stub. It should be deleted within 90 days from the current date.*
\ No newline at end of file
+*This file is a redirect stub. It should be deleted by 04/22/2019.*
```

Comment:
Can we have all the dates to match a single date? It'll be easier to create a PR then to remove all of them at once. :)

Reply:
Sure!

Reply:
Done